python/auxiliary.py

```diff
@@ -202,7 +203,7 @@ def get_moments(data):
 
     # Save mean and standard deviation of wages for each period
     # to Wage Distribution section of the moments dictionary
-    for period in range(30):  ## TO DO: Remove hard coded number
+    for period in range(30):  # TO DO: Remove hard coded number
```

Comment:
as above, please either endogenize or move to the top of the module.

Reply:
As above